src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements Drawable, AreaCalculator {
```

Comment:
According to the checklist, do not use abstract classes to set behavior for classes. Consider using an interface to define behaviors and use concrete classes to implement them.